.php_cs.dist

```diff
@@ -52,7 +52,7 @@ return PhpCsFixer\Config::create()
         'ordered_class_elements' => true,
         'ordered_imports' => true,
         'php_unit_construct' => true,
-        'php_unit_dedicate_assert' => true,
+        'php_unit_dedicate_assert' => false,
```

Comment:
Because assertInternalType is deprecated

Reply:
Just delete this line.